app/src/main/java/com/bogdan/codeforceswatcher/feature/users/redux/UsersState.kt

```diff
@@ -0,0 +1,38 @@
+package com.bogdan.codeforceswatcher.feature.users.redux
+
+import com.bogdan.codeforceswatcher.model.User
+import org.rekotlin.StateType
+
+data class UsersState(
+    val status: Status = Status.IDLE,
+    val users: List<User> = listOf(),
+    val result: List<Pair<String, Int>> = listOf(),
+    val sortType: SortType = SortType.DEFAULT
+) : StateType {
+
+    enum class Status { IDLE, PENDING }
+    enum class SortType {
+        DEFAULT, RATING_UP, RATING_DOWN, UPDATE_UP, UPDATE_DOWN;
+
+        companion object {
+            fun getPositionFromSortType(sortType: SortType) =
+                when (sortType) {
+                    DEFAULT -> 0
+                    RATING_DOWN -> 1
+                    RATING_UP -> 2
+                    UPDATE_DOWN -> 3
+                    UPDATE_UP -> 4
+                }
+
+            fun getSortTypeFromPosition(sortType: Int) =
```

Comment:
Current name is little bit misleading, so `fun getSortType(position: Int)` would be clearer.